lib/engine/game/g_2038/company.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require_relative '../../company'
+
+module Engine
+  module Game
+    module G2038
+      class Company < Engine::Company
```

Comment:
why do you need this class? i'd remove it

Reply:
This class is currently used to get the initial draft to work right.  If I remove it the project won't compile anymore.

Reply:
just delete this class, and then comment out the failing code in the step